server/src/main/java/org/elasticsearch/index/mapper/IpFieldMapper.java

```diff
@@ -443,34 +447,28 @@ protected String contentType() {
 
     @Override
     protected void parseCreateField(DocumentParserContext context) throws IOException {
```

Comment:
I wanted very much to reuse the parsing code but it was super twisty. I believe the new code doesn't change the behavior but let's us reuse it during tsid extraction.